src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3985,6 +4079,109 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if ( FlagHybridModel && !WarmupFlag && !DoingSizing ) {
+
+				Zone( ZoneNum ).ZoneMeasuredTemperature = GetCurrentScheduleValue( HybridModelZone( ZoneNum ).ZoneMeasuredTemperatureSchedulePtr );
+
+				// HM calculation only HM calculation period start
+				if ( DayOfYear >= HybridModelZone( ZoneNum ).HybridStartDayOfYear && DayOfYear <= HybridModelZone( ZoneNum ).HybridEndDayOfYear ){
+
+					Real64 HMMultiplierAverage( 1.0 );
+					Real64 MultpHM( 1.0 );
+					Real64 InfilOAACHHM( 0.0 );
+
+					ZT( ZoneNum ) = Zone( ZoneNum ).ZoneMeasuredTemperature;
+
+					// Hybrid Modeling-Infiltration calcualtion start
+					if ( HybridModelZone( ZoneNum ).InfiltrationCalc && SumSysMCpT == 0 && UseZoneTimeStepHistory ){ // HM calculation only when SumSysMCpT =0, TimeStepZone (not @ TimeStepSys)
+
+						// Calculate MCPI used for hybrid modeling included in TempIndCoef and TempDepCoef
+						Real64 a = Zone( ZoneNum ).OutDryBulbTemp;
+						Real64 b = SumIntGain + SumHATsurf - SumHATref + MCPTV( ZoneNum ) + MCPTM( ZoneNum ) + MCPTE( ZoneNum ) + MCPTC( ZoneNum ) + MDotCPOA( ZoneNum ) * Zone( ZoneNum ).OutDryBulbTemp;
+						Real64 c = SumHA + MCPV( ZoneNum ) + MCPM( ZoneNum ) + MCPE( ZoneNum ) + MCPC( ZoneNum ) + MDotCPOA( ZoneNum );
+						Real64 d = AirCap;
+						//Use3rdOrder
+						Real64 AA = b + d * ( 3.0 * PreviousMeasuredZT1( ZoneNum ) - ( 3.0 / 2.0 ) * PreviousMeasuredZT2( ZoneNum ) + ( 1.0 / 3.0 ) * PreviousMeasuredZT3( ZoneNum ) );
+						Real64 BB = ( 11.0 / 6.0 ) * d + c;
+						Real64 InfilVolumeOADensityHM;
+						Real64 AirDensity;
+						Real64 CpAir;
+						Real64 MCPIHM;
+						static std::string const RoutineNameInfiltration( "CalcAirFlowSimple:Infiltration" );
+
+     					CpAir = PsyCpAirFnWTdb( OutHumRat, Zone( ZoneNum ).OutDryBulbTemp );
+						AirDensity = PsyRhoAirFnPbTdbW( OutBaroPress, Zone( ZoneNum ).OutDryBulbTemp, OutHumRat, RoutineNameInfiltration );
+						MCPIHM = ( AA - Zone( ZoneNum ).ZoneMeasuredTemperature * BB ) / ( Zone( ZoneNum ).ZoneMeasuredTemperature - a ); // MCPIHM calculation using Use3rdOrder method 
```

Comment:
This could cause a divide by zero if .ZoneMeasuredTemperature = a = .OutDryBulbTemp.  Should protect this divide and set MCPIHM to zero if the deltaT is not > say 0.001 or so.